### PR TITLE
Create new product docs release note

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

This pull request captures the changes to the DevHub/docs.getdbt.com over the last 4ish weeks.

## Checklist

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "April 2023 product docs updates"
+id: "april-product-docs"
+description: "April 2023: "
+sidebar_label: "Update: product docs updates"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]: this is a personal pref - i feel it reads weird when the rn labele is update and then update is written again. another suggestion is 'product docs patches'  or 'product docs tweaks'. 

```suggestion
sidebar_label: "Update: product docs changes"
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+sidebar_position: 8
+tags: [Apr-2023, product-docs]
+---
+
+Here are some docs.getdbt.com highlights from the last few weeks:
+
+- [API docs](/docs/dbt-cloud-apis/overview) now live in the left sidebar to improve discoverability.
+- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info on the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provides everyone with guidance wrt deploying with other tools.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.
```

**Comment by mirnawong1** [comment]
this looks awesome @runleonarun - i'm wondering:

- should we use this as an opportunity to softly introduce yourself and the docs team? maybe it's not necessary or maybe you have it planned later but just a question because thought we can use it to post in the community.

- also i wonder if the release note should have h2 to break down the efforts (e.g. discoverability, cloud projects, core products, guides/blogs). for example:

Here are some docs.getdbt.com highlights from the last few weeks:

## 🔎 Discoverability

- [API docs](/docs/dbt-cloud-apis/overview) now live in the left sidebar to improve discoverability.
- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.

## ☁ Cloud projects

- Added Starburst/Trino adapter docs, including:
  * [dbt Cloud quickstart guide](/docs/quickstarts/dbt-cloud/starburst-galaxy), 
  * [connection page](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-starburst-trino), 
  * [set up page](https://docs.getdbt.com/reference/warehouse-setups/trino-setup), and [config page](https://docs.getdbt.com/reference/resource-configs/trino-configs). 
- Enhanced [dbt Cloud jobs page](https://docs.getdbt.com/docs/deploy/dbt-cloud-job) and section to include conceptual info on the queue time, improvements made around it, and about failed jobs. 
- Check out the April dbt [Cloud release notes](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes)

## 🎯 Core projects 

- Clearer descriptions in the [Jinja functions page](/reference/dbt-jinja-functions), that improve content for each card. 
- [1.5 Docs](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.5) have been released as an RC! 
- See the beautiful [work captured in Core v 1.5](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.5%22+is%3Aclosed).

## New 📚 Guides and ✏️ blog posts

- [Use Databricks workflows to run dbt Cloud jobs](/guides/orchestration/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs)
- [Refresh Tableau workbook with extracts after a job finishes](/guides/orchestration/webhooks/zapier-refresh-tableau-workbook)
- [dbt Python Snowpark workshop/tutorial](https://docs.getdbt.com/guides/dbt-ecosystem/dbt-python-snowpark/1-overview-dbt-python-snowpark)
- [How to optimize and troubleshoot dbt Models on Databricks](https://docs.getdbt.com/guides/dbt-ecosystem/databricks-guides/how_to_optimize_dbt_models_on_databricks)
- [The missing guide to debug() in dbt](https://docs.getdbt.com/blog/guide-to-jinja-debug)
- [dbt Squared: Leveraging dbt Core and dbt Cloud together at scale](https://docs.getdbt.com/blog/dbt-squared)
- [Audit_helper in dbt: Bringing data auditing to a higher level](https://docs.getdbt.com/blog/audit-helper-for-migration)

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "April 2023 product docs updates"
+id: "april-product-docs"
+description: "April 2023: "
+sidebar_label: "Update: product docs changes"
+sidebar_position: 8
+tags: [Apr-2023, product-docs]
+---
+
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md:9]: Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! We want to share some highlights introduced to docs.getdbt.com in the last month:

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Here are some docs.getdbt.com highlights from the last few weeks:
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! We want to share some highlights introduced to docs.getdbt.com in the last month:
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+- Clearer descriptions in the [Jinja functions page](/reference/dbt-jinja-functions), that improve content for each card. 
+- [1.5 Docs](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.5) have been released as an RC! 
+- See the beautiful [work captured in Core v 1.5](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.5%22+is%3Aclosed).
+
+## New 📚 Guides and ✏️ blog posts
+
+- [Use Databricks workflows to run dbt Cloud jobs](/guides/orchestration/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs)
+- [Refresh Tableau workbook with extracts after a job finishes](/guides/orchestration/webhooks/zapier-refresh-tableau-workbook)
+- [dbt Python Snowpark workshop/tutorial](https://docs.getdbt.com/guides/dbt-ecosystem/dbt-python-snowpark/1-overview-dbt-python-snowpark)
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
- [dbt Python Snowpark workshop/tutorial](/guides/dbt-ecosystem/dbt-python-snowpark/1-overview-dbt-python-snowpark)
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- [How to optimize and troubleshoot dbt Models on Databricks](https://docs.getdbt.com/guides/dbt-ecosystem/databricks-guides/how_to_optimize_dbt_models_on_databricks)
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
- [How to optimize and troubleshoot dbt Models on Databricks](/guides/dbt-ecosystem/databricks-guides/how_to_optimize_dbt_models_on_databricks)
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  * [dbt Cloud quickstart guide](/docs/quickstarts/dbt-cloud/starburst-galaxy), 
+  * [connection page](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-starburst-trino), 
+  * [set up page](https://docs.getdbt.com/reference/warehouse-setups/trino-setup), and [config page](https://docs.getdbt.com/reference/resource-configs/trino-configs). 
+- Enhanced [dbt Cloud jobs page](https://docs.getdbt.com/docs/deploy/dbt-cloud-job) and section to include conceptual info on the queue time, improvements made around it, and about failed jobs. 
+- Check out the April dbt [Cloud release notes](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes)
+
+## 🎯 Core projects 
+
+- Clearer descriptions in the [Jinja functions page](/reference/dbt-jinja-functions), that improve content for each card. 
+- [1.5 Docs](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.5) have been released as an RC!
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
- [1.5 Docs](/guides/migration/versions/upgrading-to-v1.5) have been released as an RC! 
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+- [API docs](/docs/dbt-cloud-apis/overview) now live in the left sidebar to improve discoverability.
+- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.
+
+## ☁ Cloud projects
+
+- Added Starburst/Trino adapter docs, including:
+  * [dbt Cloud quickstart guide](/docs/quickstarts/dbt-cloud/starburst-galaxy), 
+  * [connection page](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-starburst-trino), 
+  * [set up page](https://docs.getdbt.com/reference/warehouse-setups/trino-setup), and [config page](https://docs.getdbt.com/reference/resource-configs/trino-configs).
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
  * [set up page](/reference/warehouse-setups/trino-setup), and [config page](https://docs.getdbt.com/reference/resource-configs/trino-configs). 
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+- [API docs](/docs/dbt-cloud-apis/overview) now live in the left sidebar to improve discoverability.
+- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.
+
+## ☁ Cloud projects
+
+- Added Starburst/Trino adapter docs, including:
+  * [dbt Cloud quickstart guide](/docs/quickstarts/dbt-cloud/starburst-galaxy), 
+  * [connection page](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-starburst-trino), 
+  * [set up page](https://docs.getdbt.com/reference/warehouse-setups/trino-setup), and [config page](https://docs.getdbt.com/reference/resource-configs/trino-configs). 
+- Enhanced [dbt Cloud jobs page](https://docs.getdbt.com/docs/deploy/dbt-cloud-job) and section to include conceptual info on the queue time, improvements made around it, and about failed jobs.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
- Enhanced [dbt Cloud jobs page](/docs/deploy/dbt-cloud-job) and section to include conceptual info on the queue time, improvements made around it, and about failed jobs. 
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+## 🔎 Discoverability
+
+- [API docs](/docs/dbt-cloud-apis/overview) now live in the left sidebar to improve discoverability.
+- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.
+
+## ☁ Cloud projects
+
+- Added Starburst/Trino adapter docs, including:
+  * [dbt Cloud quickstart guide](/docs/quickstarts/dbt-cloud/starburst-galaxy), 
+  * [connection page](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-starburst-trino),
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
  * [connection page](/docs/cloud/connect-data-platform/connect-starburst-trino), 
```

**Review by mirnawong1 — DISMISSED** [review]

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+- [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.
+
+## ☁ Cloud projects
+
+- Added Starburst/Trino adapter docs, including:
+  * [dbt Cloud quickstart guide](/docs/quickstarts/dbt-cloud/starburst-galaxy), 
+  * [connection page](/docs/cloud/connect-data-platform/connect-starburst-trino), 
+  * [set up page](/reference/warehouse-setups/trino-setup), and [config page](https://docs.getdbt.com/reference/resource-configs/trino-configs). 
+- Enhanced [dbt Cloud jobs page](/docs/deploy/dbt-cloud-job) and section to include conceptual info on the queue time, improvements made around it, and about failed jobs. 
+- Check out the April dbt [Cloud release notes](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes)
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md]:
```suggestion
- Check out the April dbt [Cloud release notes](/docs/dbt-versions/dbt-cloud-release-notes)
```

**Review by mirnawong1 — CHANGES_REQUESTED** [review]

**Review by mirnawong1 — APPROVED** [review]
🚢 @runleonarun

**Pull request merged**